### ISSUE TITLE
Loss function converges

### ISSUE THREAD

**Issue by guoees** [Arthur151/ROMP#274]
hello,The loss function converges relatively high, and I want to know how much the loss function converges to a better model.

**Comment by Arthur151** [comment]
Hi, @guoees .
Yes, the training process needs quite some time. 
Which model are you triaining, ROMP or BEV?  
For ROMP, the overall loss is supposed to converge to ~ 120.  
For BEV, the overall loss is supposed to converge to ~ 350.  
Training more time would be very benefical to the performance.
